app/models/cloud_tenant.rb

```diff
@@ -5,7 +5,7 @@ class CloudTenant < ApplicationRecord
   include NewWithTypeStiMixin
   extend ActsAsTree::TreeWalker
 
-  belongs_to :ext_management_system, :foreign_key => "ems_id", :class_name => "ManageIQ::Providers::CloudManager"
+  belongs_to :ext_management_system, :foreign_key => "ems_id", :class_name => "ExtManagementSystem"
```

Comment:
You should be able to drop the `:class_name` since you're targeting all descendants, right?

Reply:
done